EIPS/Skinny_CREATE2.md

```diff
@@ -0,0 +1,16 @@
+```
+EIP: <to be assigned>
+Title: Skinny CREATE2
+Author: Vitalik Buterin
```

Comment:
Please include a username or email address (parentheses for Github username) or <triangle brackets for email>.

Reply:
lol. @vbuterin please follow protocol around here... just because you created Ethereum doesn't mean you get to break EIP specification.